test/index.js

```diff
@@ -26,739 +26,615 @@ describe('Token', function () {
+      await newServer.register(require('../'))
+      newServer.auth.strategy('default', 'jwt', { key: privateKey, validateFunc: loadUser, audience: [audience, 'audience2', 'audience3']});
+      newServer.auth.default('default');
 
-      server.route([
+      newServer.route([
         { method: 'POST', path: '/token', handler: tokenHandler, config: { auth: 'default' } }
       ]);
+
+      await newServer.start();
     });
-    
-    it('fails if token audience is empty', function (done) {
+
+    it('fails if token audience is empty', async function () {
       var request = { method: 'POST', url: '/token', headers: { authorization: tokenHeader('john') } };
 
-      server.inject(request, function (res) {
-        expect(res.result.message).to.equal(jwtErrorPrefix + 'jwt audience invalid. expected: ' + audience + ' or audience2 or audience3');
-        expect(res.statusCode).to.equal(401);
-        done();
-      });
+      const res = await newServer.inject(request);
+      expect(res.result.message).to.equal(jwtErrorPrefix + 'jwt audience invalid. expected: ' + audience + ' or audience2 or audience3');
+      expect(res.statusCode).to.equal(401);
     });
 
-    it('fails if token audience is invalid', function (done) {
-  
+    it('fails if token audience is invalid', async function () {
       var request = { method: 'POST', url: '/token', headers: { authorization: tokenHeader('john', {audience:'https://invalid.audience.com'}) } };
 
-      server.inject(request, function (res) {
-        expect(res.result.message).to.equal(jwtErrorPrefix + 'jwt audience invalid. expected: ' + audience + ' or audience2 or audience3');
-        expect(res.statusCode).to.equal(401);
-        done();
-      });
+      const res = await newServer.inject(request);
+      expect(res.result.message).to.equal(jwtErrorPrefix + 'jwt audience invalid. expected: ' + audience + ' or audience2 or audience3');
+      expect(res.statusCode).to.equal(401);
     });
 
-    it('works if token audience is one of the expected values', function (done) {
-  
+    it('works if token audience is one of the expected values', async function () {
       var request = { method: 'POST', url: '/token', headers: { authorization: tokenHeader('john', {audience: audience}) } };
 
-      server.inject(request, function (res) {
-        expect(res.result).to.exist;
-        expect(res.statusCode).to.equal(200);
-        done();
-      });
+      const res = await newServer.inject(request);
+      expect(res.result).to.exist;
+      expect(res.statusCode).to.equal(200);
     });
 
   });
 
   describe('when a single issuer is specified for validation', function(){
-    var issuer = 'http://expected.issuer'; 
-    
-    var server = new Hapi.Server({ debug: false});
-    server.log(['error', 'database', 'read']);
-    server.connection();
-    server.register(require('../'), function (err) {
-      expect(err).to.not.exist;
+    var issuer = 'http://expected.issuer';
```

Comment:
We removed the `.log` call here, but kept it on the following one.

Reply:
Removed